frontend/src/scenes/settings/SettingsMap.tsx

```diff
@@ -240,6 +241,12 @@ export const SETTINGS_MAP: SettingSection[] = [
                 id: 'replay-authorized-domains',
                 title: 'Authorized domains for replay',
                 component: <ReplayAuthorizedDomains />,
+                allowForTeam: (t) => !!t?.recording_domains?.length,
```

Comment:
opted to add this check rather than create a feature flag manually to cover 1000s of teams

Reply:
nice